leapp/models/fields/__init__.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, default=None, help=None):  # noqa; pylint: disable=redefined-
         self._nullable = False
         self._default = default
 
-        if type(self) == Field:
+        if type(self) == Field:  # noqa;
```

Comment:
Hmm maybe `is` comparison could work here?

Reply:
let me try

Reply:
@fernflower  you were right!! thanks! updated